test-e2e/build/no-deps-_build.test.js

```diff
@@ -7,7 +7,8 @@ const {initFixture} = require('../test/helpers');
 it('Build - no deps _build', async () => {
   expect.assertions(1);
   const p = await initFixture(path.join(__dirname, './fixtures/no-deps-_build'));
+  await p.esy('install');
```

Comment:
Unfortunately, `esy install` is currently blocked on Windows (due to #302 ). We can use `legacy-install` on Windows for now, until we unblock the `esyi` flow. We might want to add this as a helper in `initFIxture` like `p.esyInstall()`, or as an `InstallCommand` constant we can switch conditionally for windows vs OSX/Mac.

Reply:
Added this in helpers.js:
```js
    // workaround until https://github.com/esy/esy/issues/302 is fixed
    if (args === 'install' && os.platform() === 'win32') {
      args = 'legacy-install';
    }
```